infrastructure/quick-deploy/localhost/all/variables.tf

```diff
@@ -0,0 +1,383 @@
+# Kubeconfig path
+variable "k8s_config_path" {
+  description = "Path of the configuration file of K8s"
+  type        = string
+  default     = "~/.kube/config"
+}
+
+# Kubeconfig context
+variable "k8s_config_context" {
+  description = "Context of K8s"
+  type        = string
+  default     = "default"
+}
+
+# Prefix
+variable "prefix" {
+  description = "Prefix used to name all the resources"
+  type        = string
+  default     = null # random
+}
+
+# Kubernetes namespace
+variable "namespace" {
+  description = "Kubernetes namespace for ArmoniK"
+  type        = string
+  default     = "armonik"
+}
+
+# Logging level
+variable "logging_level" {
+  description = "Logging level in ArmoniK"
+  type        = string
+  default     = "Information"
+}
+
+# Metrics Server
+variable "metrics_server" {
+  description = "Parameters of the metrics server"
+  type = object({
+    namespace          = optional(string, "kube-system"),
+    image_name         = optional(string, "k8s.gcr.io/metrics-server/metrics-server"),
+    image_tag          = optional(string, "v0.6.1"),
+    image_pull_secrets = optional(string, ""),
+    node_selector      = optional(any, {}),
+    args = optional(list(string), [
+      "--cert-dir=/tmp",
+      "--kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname",
+      "--kubelet-use-node-status-port",
+      "--metric-resolution=15s",
+    ]),
+    host_network = optional(bool, false),
+  })
```

Comment:
I really like those optional !

Reply:
Me too, I'm just unsure about putting image tags in the default value...

Reply:
I understand your point but we do not really have a proper alternative to set up those tags at one place. Maybe putting them in the module by default, but it won't really solve the issue.

Reply:
versions.tf would have been great